src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx

```diff
@@ -15,7 +15,7 @@ export default function ReviewFieldTemplate(props) {
   return schema.type === 'object' || schema.type === 'array' ? (
     children
   ) : (
-    <div className="review-row">
+    <dl className="review-row">
```

Comment:
I did some more checking on localhost, and I think the fix is one level higher. This change to a `<dl>` from a `<div>` causes the table to be made up of many one-item definition lists, which is arguably worse user experience.

I did a lot of looking at the code in localhost v. staging, and noticed the issue was some of the `<div className="review-row">` had another unclassed `<div>` wrapping them. Removing that non-specific `<div>` removed the axe error. Screenshot attached.

---
![Screen Shot 2019-11-19 at 9 21 32 AM](https://user-images.githubusercontent.com/934879/69160333-03f27f80-0aaf-11ea-9ba2-90746b66d7e8.png)

Reply:
Thanks! I'll investigate how to remove those extra div wrappers

Reply:
Sure thing. If we're running React 16.8 or later, might be able to use `React.Fragment` ?

Reply:
All fixed, please test again... in the latest React, we can use the "shortcut" of `<>... </>` instead of `React.Fragment`

Reply:
The main problem was the ReviewCardField component wrapping the components in a div.